src/main/fc/config.c

```diff
@@ -419,9 +419,11 @@ static void validateAndFixConfig(void)
 #endif
 
 #if defined(USE_BEEPER)
+#ifdef USE_TIMER
```

Comment:
Not sure why this is needed - this disables the PWM beeper if no pin is defined.

Reply:
It's to avoid `timerGetByTag` from being called. PWM doesn't work if `USE_TIMER` is not in effect anyways.